src/main/java/org/folio/edgecommonspring/util/PropertiesUtil.java

```diff
@@ -0,0 +1,45 @@
+package org.folio.edgecommonspring.util;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Properties;
+import java.util.regex.Pattern;
+import lombok.extern.log4j.Log4j2;
+import org.folio.edgecommonspring.exception.AuthorizationException;
+
+@Log4j2
+public class PropertiesUtil {
```

Comment:
The same. I think static methods can be moved to edge-api-utils. Especially, it they can be used in both edge-common and edge-common-spring.

Reply:
Yes, this we can, but need to ask Andrii. Do we need to this util for edge-common or it's only for spring way

Reply:
we can move this class

Reply:
Wait a second, guys. I understand this class is part of `EdgeVerticleCore` from `edge-common`. `edge-common` return `Future<Properties>` type and `edge-common-spring` - just `properties.

Won't this be a problem?

Reply:
oh, yes, then don`t need to move it to the api-utils

Reply:
I agree that this too can and probably should move to edge-api-utils.  Again, edge-common will need to make some adjustments to adopt edge-api-utils.  I think that's expected, and acceptable.

Reply:
updated

Reply:
moved